docs/modules/reference/nav.adoc

```diff
@@ -14,4 +14,17 @@
 ** xref:configuration/tuning-activemq.adoc[]
 ** xref:configuration/ttl-rpc.adoc[]
 ** xref:configuration/aws-sqs.adoc[]
+** xref:configuration/telemetryd/listener/introduction.adoc[]
```

Comment:
I would suggest moving these all up to the root level of Reference, instead of being nested under Configuration.

Reply:
```
* xref:configuration/telemetryd/introduction.adoc[]
** Listeners
*** xref:configuration/telemetryd/listener/tcp.adoc[]
*** xref:configuration/telemetryd/listener/udp.adoc[]
** Protocols
*** xref:configuration/telemetryd/protocols/bmp.adoc[]
*** xref:configuration/telemetryd/protocols/nxos.adoc[]
*** xref:configuration/telemetryd/protocols/graphite.adoc[]
*** xref:configuration/telemetryd/protocols/ipfix.adoc[]
*** xref:configuration/telemetryd/protocols/jti.adoc[]
*** xref:configuration/telemetryd/protocols/netflow5.adoc[]
*** xref:configuration/telemetryd/protocols/netflow9.adoc[]
*** xref:configuration/telemetryd/protocols/openconfig.adoc[]
*** xref:configuration/telemetryd/protocols/sflow.adoc[]
```

Reply:
That's a good idea.